test/metabase/db/custom_migrations/pulse_to_notification_test.clj

```diff
@@ -42,7 +52,7 @@
   (let [pulse-id (t2/insert-returning-pk! :pulse (add-timestamp pulse))]
     (t2/insert! :pulse_card (map #(assoc % :pulse_id pulse-id) pulse-cards))
     (doseq [pcr pcs+recipients]
-      (let [pc-id (t2/insert-returning-pk! :pulse_channel (-> pcr (assoc :pulse_id pulse-id) (dissoc :recipients) add-timestamp))]
+      (let [pc-id (t2/insert-returning-pk! :model/PulseChannel (-> pcr (assoc :pulse_id pulse-id) (dissoc :recipients) add-timestamp))]
```

Comment:
this pulse channel hooks have sideaffects to create quartz triggers that we want to test